configure.ac

```
@@ -1,5 +1,5 @@
 AC_PREREQ([2.60])
-AC_INIT([libdashbls],[1.3.0])
+AC_INIT([libdashbls],[1.3.1])
```

we have 1.3.1 already https://github.com/dashpay/bls-signatures/releases/tag/v1.3.1
```suggestion
AC_INIT([libdashbls],[1.3.2])
```

---

oups